spec/01-abstract.md

```diff
@@ -1,7 +1,39 @@
-Distributed tracing is a set of tools and practices to monitor the health and reliability of a distributed application. A distributed application is an application that consists of multiple components that are deployed and operated separately. It is also known as micro-service.
+# Problem Statement
 
-The main concept behind distributed tracing is event correlation. Event correlation is a way to correlate events from one component to the events from another. It allows to find the cause-and-effect relationship between these events. For instance – find which user action in a browser caused a failure in the business logic layer.
+Distributed tracing is a methodology implemented by tracing tools to follow, analyze
+and debug a transaction across multiple software components. Typically, a trace
+traverses more than one component which requires it to be uniquely identifiable
+across all participating systems. Passing along this unique identification is
+provided by *trace context propagation*.

```

Comment:
I don't understand the part "is provided by trace context propagation" part of a sentence. Perhaps "provided" is not the right word here. What does this sentence mean?

Reply:
This has been changed by request. If I remember correctly @yurishkuro was involved in this discussion.
Before it was phrased ''is called trace context propagation".
This was considered to be incorrect.

Trace context propagation provides the functionality to pass along a unique identifier.
Please suggest another wording if you feel this is incorrect.

Reply:
got it. "called" instead of "provided by" makes sentence understandable. But I can see where @yurishkuro is coming from. Perhaps this feedback is similar to feedback I gave further below (not sure):

> scoping this work down to unique identifier of the trace is very limiting. Why the original paragraph about cause-and-effect events correlation wasn't good enough?

Perhaps you can reuse parts of an original abstract for better definitions.

Reply:
The original paragraph introduced a lot of new concepts and was rather confusing.